packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -61,34 +76,43 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
   });
   if (!loggedInUser) throw new TRPCError({ code: "UNAUTHORIZED", message: "You are not authorized." });
 
-  const IS_USER_ADMIN = loggedInUser.role === UserPermissionRole.ADMIN;
+  return loggedInUser;
+};
 
-  if (!ORG_SELF_SERVE_ENABLED && !IS_USER_ADMIN && !isPlatform) {
+const checkUserIsAdminOrThrow = (
```

Comment:
It isn't just about Admin. It is checking the permission to create the organization.

So, how about `ensurePermission()` , `throwIfNotAllowedToCreateOrg`